mmv1/products/networkservices/EndpointPolicy.yaml

```diff
@@ -0,0 +1,152 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'EndpointPolicy'
+base_url: 'projects/{{project}}/locations/global/endpointPolicies'
```

Comment:
The API documentation implies the location's configurable. Can you link documentation that indicates this is global-only?

Reply:
Yes, please check the "name" section here: https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1/projects.locations.endpointPolicies

Reply:
Ah, I see on https://cloud.google.com/traffic-director/docs/reference/network-services/rest/v1/projects.locations.endpointPolicies/create that `global` is fixed as well. Thanks!